src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiExport.tsx

```diff
@@ -0,0 +1,443 @@
+                          </Form.Item>
+                        </Col>
+                        <Col md={12} xs={24}>
+                          <Form.Item
+                            name="namespace"
+                            rules={[
+                              {
+                                required: true,
+                                message: i18n(
+                                  "NcbiSubmissionRequest.namespace.description"
+                                ),
+                              },
+                            ]}
+                            label={i18n("NcbiSubmissionRequest.namespace")}
+                            help={i18n(
+                              "NcbiSubmissionRequest.namespace.description"
+                            )}
+                          >
+                            <Input />
+                          </Form.Item>
+                        </Col>
+                        <Col md={12} xs={24}>
+                          <Form.Item
+                            name="releaseDate"
+                            rules={[
+                              {
+                                required: true,
+                                message: i18n(
+                                  "NcbiSubmissionRequest.releaseDate.description"
+                                ),
+                              },
+                            ]}
+                            label={i18n("NcbiSubmissionRequest.releaseDate")}
+                            help={i18n(
+                              "NcbiSubmissionRequest.releaseDate.description"
+                            )}
+                          >
+                            <DatePicker
+                              style={{ width: "100%" }}
+                              disabledDate={disabledDate}
+                            />
+                          </Form.Item>
+                        </Col>
+                      </Row>
+                    </Card>
+                    <Card title={i18n("CreateNcbiExport.samples")}>
+                      {invalid.length > 0 && (
+                        <Alert
+                          type="warning"
+                          showIcon
+                          message={"Some Samples are not valid"}
+                          closable
+                          onClose={removeInvalidSamples}
+                        />
+                      )}
+                      <CreateNcbiDefaultOptions onChange={updateDefaultValue} />
+                      <CreateNcbiExportSamples
+                        form={form}
+                        samples={samples}
+                        removeSample={removeSample}
+                      />
+                    </Card>
+
+                    {buttons}
+                  </Space>
+                </Form>
+              </>
+            ) : (
+              <Alert
+                type="error"
+                showIcon
+                message={"NO VALID SAMPLES"}
```

Comment:
Internationalize this text and the text below for the description

Reply:
Updated in a17f636510f4e0944245525373423556ab034d51